builder/googlecompute/config.go

```diff
@@ -231,6 +232,13 @@ func NewConfig(raws ...interface{}) (*Config, []string, error) {
 		errs = packer.MultiErrorAppend(fmt.Errorf("you may not specify a 'service_account_email' when 'disable_default_service_account' is true"))
 	}
 
+	if c.StartupScriptFile != "" {
+		if _, err := os.Stat(c.StartupScriptFile); err != nil {
+			errs = packer.MultiErrorAppend(
+				errs, fmt.Errorf("startup_script_file not found: %s", c.StartupScriptFile))
```

Comment:
Hi @azr, thanks for the feedback. The error message format you specified would return `startup_script_file: no-such-file stat no-such-file: no such file or directory`. 
This seems a bit confusing for users.
 Let me know if you would like me to still make the change. Thanks!

Reply:
`startup_script_file: ${file} stat ${file}: no such file or directory` is repetitive in this case; you are right, and I didn't check before but the doc of os.Stat says: 
> If there is an error, it will be of type *PathError.

Then in that case I'd return the error as is: `packer.MultiErrorAppend(errs, err)` what do you think ?

Reply:
I'd say alternative messages happen because a lot of different people contribute and review.
Personally, I like Dave Cheney's philosophy in terms of [error handling](https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully).

The way you did it looks good to me. 🙂

Thanks for contributing !!